.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @sonarsource/languages-team
```

Comment:
This belongs to the sonarqube team (a choice had to be made between SQ and SC team)  as discussed with @philippe-perrin-sonarsource

Reply:
@julien-carsique-sonarsource This is shared repo between the SonarQube and the SonarCloud team. After discussing it with @gregaubert, it makes more sense for SonarCloud to be the owner.